FSharp.AutoComplete/Program.fs

```diff
@@ -607,6 +611,21 @@ module internal Main =
 
             main state
 
+
+          | Methods ->
+            let meth = tyRes.GetMethods(line, col, lineStr)
+                       |> Async.RunSynchronously
+            match meth with
+            | Some (name,info) ->
```

Comment:
info.Length should be greater than 0 (https://github.com/fsharp/xamarin-monodevelop-fsharp-addin/blob/master/MonoDevelop.FSharpBinding/FSharpTextEditorCompletion.fs#L351)